src/catalog/database_catalog.cpp

```diff
@@ -1610,6 +1642,11 @@ void DatabaseCatalog::BootstrapTypes(transaction::TransactionContext *const txn)
              postgres::Type::BASE);
 }
 
+void DatabaseCatalog::BootstrapLanguages(transaction::TransactionContext *const txn)
+{
+  InsertLanguage(txn, "plpgsql");
```

Comment:
This breaks the current catalog convention by assigning a "userspace" OID value to an internal catalog object.  The OID number should be a constant defined in `pg_language.h` and forced during bootstrap.  It may be useful to pull most of the table logic in `InsertLanguage` into a private helper that takes the OID as a parameter so that you can use it during bootstrap and still provide the OID-free interface for the main API.